mobile-app/src/components/HeaderOptions.tsx

```diff
@@ -207,7 +218,49 @@ export function editProfileOptions(
       <RoundedButton
         text="Done"
         icon={IconCheck}
-        onPress={() => {
+        onPress={async () => {
```

Comment:
make a func, move to api.ts (look at https://github.com/owengretzinger/habits-together/blob/firebase-integration/mobile-app/src/firebase/api.ts to see get a general vibe of how we doing this Firebase stuff)